stackit-state-bucket/tests/state_bucket.tftest.hcl

```
@@ -53,9 +56,40 @@ run "state_outputs" {
     error_message = "Backend file content is incorrect"
   }
   assert {
-    condition     = nonsensitive(output.envrc_file) == "export AWS_ACCESS_KEY_ID=\"mock-access-key\"\nexport AWS_SECRET_ACCESS_KEY=\"mock-secret-key\"\n"
+    condition     = output.envrc_file == "export STACKIT_SERVICE_ACCOUNT_KEY=\"op://Employee/STACKIT Terraform Credentials/notesPlain\"\n\nexport AWS_ACCESS_KEY_ID=\"op://Employee/test-bucket-default credentials/ACCESS_KEY_ID\"\nexport AWS_SECRET_ACCESS_KEY=\"op://Employee/test-bucket-default credentials/SECRET_ACCESS_KEY\"\n"
     error_message = "Envrc file content is incorrect"
+  }
 
+  assert {
+    condition     = nonsensitive(output.onepassword_command) == "op item create --category 'Secure Note' --title 'test-bucket-default credentials' 'ACCESS_KEY_ID[text]=mock-access-key' 'SECRET_ACCESS_KEY[text]=mock-secret-key'"
+    error_message = "The command to create the 1Password item is incorrect"
   }
 }
 
+run "plan_disable_null_resources" {
+  command = plan
+
+
+  variables {
+    state_bucket_name         = "test-bucket-default"
+    write_backend_config_file = false
+    create_onepassword_item   = false
+    write_envrc_file          = false
+  }
+
+
+  assert {
+    condition     = length(null_resource.backend_config) == 0
+    error_message = "null_resource.backend_config should be planned for creation"
```

```suggestion
    error_message = "null_resource.backend_config must not be planned for creation"
```
inverted logic in error message? (same for the two assertions below)